### PR TITLE
Normalize headings and document structure.

### PR DESCRIPTION
**Pull request by jpmedley**
An equivalent PR is being made in sw-precache so that both have roughly similar section titles and order. A copy edit of the content will be in a later PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,9 +184,9 @@ Patches are encouraged, and may be submitted by forking this project and submitt
 
 Copyright 2015 Google, Inc.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+Licensed under the [Apache License, Version 2.0[(https://github.com/GoogleChrome
+/sw-toolbox/blob/master/README.md)] (the "License"); you may not use this file
```
Comment (jeffposnick) [inline README.md]: I think this should link to https://github.com/GoogleChrome/sw-toolbox/blob/master/LICENSE, which can be simplified in GitHub Markdown to `[Apache License, Version 2.0](LICENSE)`.

**Comment by jeffposnick** [comment]
@wibblymat should weigh in as well before merging, but I think this looks good other than the LICENSE link issue.

**Comment by wibblymat** [comment]
I agree about the LICENSE link, otherwise LGTM

**Comment by jpmedley** [comment]
@jeffposnick @wibblymat 

I might have jumped to a conclusion. It seedmed like both sw-toolkit and sw-precache should have the same license. As they were they both referred to Apache 2.0. 

What you requested, is this what I should do for both?

**Comment by jeffposnick** [comment]
Yep, the same change needs to be made to the `sw-precache` PR.

**Comment by jpmedley** [comment]
@jeffposnick PTAL

**Comment by jeffposnick** [comment]
:+1:

**Pull request merged**